kubectl-fdb/cmd/remove_instances.go

```diff
@@ -150,6 +155,19 @@ func getInstanceIDsFromPod(kubeClient client.Client, clusterName string, podName
 
 // removeInstances adds instances to the instancesToRemove field
 func removeInstances(kubeClient client.Client, clusterName string, instances []string, namespace string, withExclusion bool, withShrink bool, force bool) error {
+	var cluster fdbtypes.FoundationDBCluster
```

Comment:
I moved this check upfront because otherwise the user won't notice that the cluster doesn't exit.